CONTRIBUTING.md

```diff
@@ -53,24 +53,32 @@ For other ways to use `tldrl`, such as linting an entire directory, check out (w
 [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/master/pages/common/tldrl.md)
 
 ### Token syntax
-User-provided values should use the `{{token}}` syntax in order to allow clients
-to highlight them. 
+User-provided values should use the `{{token}}` syntax
+in order to allow clients to highlight them.
+Use [`snake_case`](https://en.wikipedia.org/wiki/Snake_case)
+for multi-word tokens.
 
-Some examples: 
-- `tar cf {{file}}`
-- `ln -s {{path/to/original/file}} {{path/to/link}}`
-- `mysql {{database_name}}`
-- `unrar x {{compressed.rar}}`
+Keep the following guidelines in mind when writing the tokens:
```

Comment:
I think "when choosing tokens" or "when choosing token names" works better than "writing the tokens".